src/components/Theme/src/default-theme.js

```diff
@@ -16,6 +16,12 @@ export default function defaultTheme() {
 			overlay: 'rgba(0, 0, 0, 0.3)',
 			primary: '#000000',
 		},
+		fonts: {
+			baseSize: 16,
+			sizeScale: 1.15,
+			baseLineHeight: 1.5,
+			lineHeightScale: 0.95,
```

Comment:
We currently have no plans to support either of these being values that can change at the theme level. We should avoid adding values at this level if they're not configurable.